### PR TITLE
Hotfix/v2.0.2 (develop)

### LINKED ISSUE #107: Delete account not working
Delete account is not deleting account info from Firebase

### PR DESCRIPTION
**Pull request by AlbertPG**
Closes #107

### CODE REVIEW

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/views/chart/statisticdata/StatisticData.java  1
         

Complexity decreasing per file
==============================
+ app/src/main/java/org/pesmypetcare/mypetcare/controllers/community/TrAddNewForum.java  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5593371&bid=18324659)

File: app/src/main/java/org/pesmypetcare/mypetcare/activities/views/chart/statisticdata/StatisticData.java [hunk 1]
```diff
@@ -73,17 +74,21 @@ public double getyMaxValue() {
...
         int day = dateTime.getDay();
-        StringBuilder date = new StringBuilder(String.valueOf(year));
-        date.append('-');
+        StringBuilder date = new StringBuilder("");
+
+        if (day < FIRST_TWO_DIGITS_DAY) {
+            date.append('0');
+        }
+
+        date.append(day).append("-");
```
Comment (santidrj) [inline app/src/main/java/org/pesmypetcare/mypetcare/activities/views/chart/statisticdata/StatisticData.java:85]: Codacy Issue found: [Avoid appending characters as strings in StringBuffer.append.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5593371)

**Review by santidrj — APPROVED** [review]
:+1:

**Comment by santidrj** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/main/java/org/pesmypetcare/mypetcare/activities/views/chart/statisticdata/StatisticData.java  1
         

Complexity decreasing per file
==============================
+ app/src/main/java/org/pesmypetcare/mypetcare/controllers/community/TrAddNewForum.java  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5593371&bid=18324659)

**Review by santidrj — APPROVED** [review]

**Review by AlvarTB — APPROVED** [review]

**Pull request merged**